package.json

```diff
@@ -33,11 +33,14 @@
     "core-js": "^3.6.5",
     "dotenv": "^16.0.3",
     "express": "^4.18.2",
+    "i18next": "^22.4.9",
     "lodash": "^4.17.21",
     "nanoid": "^3.3.4",
     "pixelarticons": "^1.7.0",
     "react": "18.2.0",
     "react-dom": "18.2.0",
+    "react-i18next": "^12.1.5",
+    "react-linkify": "^1.0.0-alpha",
```

Comment:
Why do we need this library?

Reply:
- react-i18next
0f9a9f9511cd39d8b5449ba15540b4d628690c4f
add UI strings to i18n to JSON file. so we don't have to handle text inside layout files. also,
we can use them to translate websites in the future.

- linkfy
this package is using for parse links on the text
![image](https://user-images.githubusercontent.com/33794760/222386257-9010143d-a211-4ff7-9333-e33c0138208b.png)
inside text component children will passed into this package. if there any links inside of the text they will wrap in `<a></a>` tags

Reply:
Removed react-linkify package (2c257c4b6d75666525cfd58fa0f67d0f5f6fdae9) and create link subcomponent to render links (Also used i18next interpolation to parse link) (2bbf2933205ac1de623519507b546aa0a7e3402e)